dataflows/processors/dumpers/file_formats.py

```diff
@@ -203,3 +203,36 @@ def write_transformed_row(self, transformed_row):
 
     def finalize_file(self):
         self.writer.write(']')
+
+
+class GeoJSONFormat(JSONFormat):
+    def __init__(self, file, schema, **options):
+        writer = file
+        writer.write('{"type": "FeatureCollection","features":')
+        writer.__first = True
+        super(GeoJSONFormat, self).__init__(writer, schema, **options)
+
+
+    def __transform_row(self, row, schema):
+        try:
+            geometry = {}
+            for k, v in row.items():
+                if schema.get_field(k).type == "geopoint":
+                    geometry = {"type": "Point",
+                                "coordinates" : self._FileFormat__transform_value(v, self.fields[k])}
```

Comment:
I think here we would like to support also the `geojson` field type.
Also I think it would be good to break here or ensure that only one geometry exists in row.

Reply:
You are right, I'll integrate that